packages/pluggableWidgets/datagrid-web/src/components/__tests__/__snapshots__/Table.spec.tsx.snap

```diff
@@ -1180,7 +1181,7 @@ exports[`Table renders the structure correctly for preview when no header is pro
           Object {
             "Cell": [Function],
             "Footer": [Function],
-            "Header": "[Empty caption]",
+            "Header": "",
```

Comment:
I feel like this change shouldn't have happened, can you double check?

Reply:
I forgot to push it before apparently, because this was changed in https://github.com/mendix/widgets-resources/pull/619/commits/1c4ed1e9eeece4f5144ee4b9fa6f917a4bac1530